scripts/send_application_on_closure.py

```diff
@@ -48,8 +49,10 @@ def send_incomplete_applications_after_deadline(fund_id, round_id, send_emails=F
             "round_id": round_id,
         }
         matching_applications = search_applications(**search_params)
+        fund_data = get_fund(fund_id)
         applications_to_send = []
         for application in matching_applications:
+            application = {**application, "fund_name": fund_data.name}
             try:
                 application["forms"] = get_forms_by_app_id(application.get("id"))
                 application["round_name"] = fund_rounds.get("title")
```

Comment:
not sure this should be in the try block given the exception handler

Reply:
additionally " fund_rounds = get_fund_round(fund_id, round_id)" could be moved within the if block to avoid a needless call. (maybe next to your new "get_fund" call?

Reply:
To be implemented in the future fix